src/code/code_800F7260.c

```diff
@@ -361,101 +360,100 @@ void func_800F7CEC(u8 arg0) {
         if (gSoundBanks[arg0][phi_s4].unk_2B == 0) {
             func_800F7B54(arg0, phi_s4);
         } else if (gSoundBanks[arg0][phi_s4].unk_2A != 0) {
-            temp_s2 = &gSoundBanks[arg0][phi_s4];
+            entry = &gSoundBanks[arg0][phi_s4];
 
-            if (&D_801333D4.x == temp_s2[0].posX) {
-                temp_s2->unk_1C = 0.0f;
+            if (&D_801333D4.x == entry[0].posX) {
+                entry->dist = 0.0f;
             } else {
-                tempf1 = *temp_s2->posY * 1;
-                temp_s2->unk_1C = (SQ(*temp_s2->posX) + SQ(tempf1) + SQ(*temp_s2->posZ)) * 1;
+                tempf1 = *entry->posY * 1;
+                entry->dist = (SQ(*entry->posX) + SQ(tempf1) + SQ(*entry->posZ)) * 1;
             }
-            spA9 = temp_s2->unk_24;
-            if (temp_s2->unk_26 & 0x10) {
-                temp_s2->unk_20 = SQ(0xFF - spA9) * 0x1690;
+            spA9 = entry->unk_24;
+            if (entry->unk_26 & 0x10) {
+                entry->unk_20 = SQ(0xFF - spA9) * 0x1690;
             } else {
-                if (temp_s2->unk_1C > 0x7FFFFFD0) {
-                    temp_s2->unk_1C = 0x70000008;
-                    osSyncPrintf(D_80133344, temp_s2->unk_28, temp_s2->posX, temp_s2->posZ, *temp_s2->posX,
-                                 *temp_s2->posY, *temp_s2->posZ);
+                if (entry->dist > 0x7FFFFFD0) {
+                    entry->dist = 0x70000008;
+                    osSyncPrintf(D_80133344, entry->unk_28, entry->posX, entry->posZ, *entry->posX,
+                                 *entry->posY, *entry->posZ);
                 }
-                temp3 = temp_s2->unk_28;
-                temp_s2->unk_20 = (u32)temp_s2->unk_1C + (SQ(0xFF - spA9) * 0x1690) + temp3 - temp3;
-                if (*temp_s2->posZ < 0.0f) {
-                    temp_s2->unk_20 += (s32)(-*temp_s2->posZ * 6.0f);
+                temp3 = entry->unk_28;
+                entry->unk_20 = (u32)entry->dist + (SQ(0xFF - spA9) * 0x1690) + temp3 - temp3;
+                if (*entry->posZ < 0.0f) {
+                    entry->unk_20 += (s32)(-*entry->posZ * 6.0f);
                 }
             }
-            if (temp_s2->unk_1C > 1.0e10f) {
-                if (temp_s2->unk_2A == 4) {
-                    Audio_QueueCmdS8((temp_s2->unk_2E << 8) | 0x6020000, 0);
-                    if (temp_s2->unk_28 & 0xC00) {
+            if (entry->dist > 1.0e10f) {
```

Comment:
```suggestion
            if (entry->dist > SQ(1.0e5f)) {
```
it's kinda weird how `entry->dist` is sometimes the square value and sometimes the actual value, in some other functions `10000 = SQ(100)`/`2500 = SQ(50)` appear near it but I can't tell if dist is squared or not at those points. Just writing for my curiosity, it can definitely wait for more documentation

Reply:
It's silly, yeah. The instances of 2500 and 10000 in code_800EC960 appear to come after `entry->dist = sqrtf(entry->dist);`.